src/styles.js

```diff
@@ -87,7 +87,6 @@ export function makeBodyVisible(doc) {
     }
   };
   interval = setInterval(set, 4);
```

Comment:
No, here we need the body opacity to be set as fast as possible. setInterval(*, 4) on iOS could take as long as 0.5 seconds to trigger the first time.